.github/workflows/publish.yaml

```diff
@@ -26,7 +26,8 @@ jobs:
           fetch-depth: 0
 
       - if: steps.decisions.outputs.PUBLISH == 'true'
-        uses: snapcore/action-build@v1
+        # XXX: waiting on a release with https://github.com/snapcore/action-build/pull/59
+        uses: snapcore/action-build@master
```

Comment:
this should be released now

Reply:
Thanks! :rocket: 

The `v1` tag is still pointing to `v1.1.2` though